src/vs/workbench/contrib/search/browser/searchActions.ts

```diff
@@ -363,6 +363,60 @@ export class CollapseDeepestExpandedLevelAction extends Action {
 	}
 }
 
+export class ExpandAllAction extends Action {
+
+	static readonly ID: string = 'search.action.expandSearchResults';
+	static LABEL: string = nls.localize('ExpandAllAction.label', "Expand All");
+
+	constructor(id: string, label: string,
+		@IViewletService private readonly viewletService: IViewletService,
+		@IPanelService private readonly panelService: IPanelService
+	) {
+		super(id, label, 'search-action codicon-chrome-restore');
```

Comment:
How can I add a new `codicon` for `expand-all` ?
(`codicon-chrome-restore` is just a placeholder)

Reply:
@misolori Can we get `expand-all` codicon? Probably just `collapse-all` but with a plus.